test/functional/test_framework/blocktools.py

```diff
@@ -196,6 +198,20 @@ def create_raw_transaction(node, txid, to_address, *, amount):
     assert_equal(signresult["complete"], True)
     return signresult['hex']
 
+def create_tx_with_large_script(prevtx, n, scriptPubKey, amt1=1, amt2=0.1 ):
+    tx = CTransaction()
+    tx.vin.append(CTxIn(COutPoint(prevtx, n), b"", 0xffffffff))
+    tx.vout.append(CTxOut(int(amt1 * COIN), scriptPubKey))
```

Comment:
Why `scriptPubkey` instead of `CScript([b'\x51'])`?

Reply:
This way we can increases the number of spendable outputs available after each transaction. There is no specific reason other than this. We can use `"datacarriermultiple"` and change this output to `OP_RETURN` too.

Reply:
Wouldn't an increase in the number of spendable outputs belonging to the wallet increase the time taken to listunspent and increase testing time?

Reply:
You are right, more utxos cause listunspent to take longer. But in this test case pruning is triggered only when 5 Mb disk space is exceeded. To build that I need a continuous supply of large transactions.
I ran the test with both options: 'scriptPubkey' and 'CScript([b'\x51'])' this the disk usage:

scriptPubkey:
``` 
2024-08-06T10:00:10.263000Z TestFramework (INFO): Warning! This test requires 4GB of disk space and takes over 30 mins (up to 2 hours)
2024-08-06T10:00:10.263000Z TestFramework (INFO): Mining a big blockchain...
2024-08-06T10:00:10.969000Z TestFramework (INFO): Disk Usage is: 17 height: 751
2024-08-06T10:07:06.738000Z TestFramework (INFO): Disk Usage is: 161 height: 2751
2024-08-06T10:18:19.266000Z TestFramework (INFO): Disk Usage is: 273 height: 4751
2024-08-06T10:32:52.381000Z TestFramework (INFO): Disk Usage is: 385 height: 6751
2024-08-06T10:51:50.736000Z TestFramework (INFO): Disk Usage is: 514 height: 8751
2024-08-06T10:54:40.091000Z TestFramework (INFO): Final Usage is: 530 height: 9000
```
CScript([b'\x51']):
``` 
2024-08-06T10:01:03.409000Z TestFramework (INFO): Warning! This test requires 4GB of disk space and takes over 30 mins (up to 2 hours)
2024-08-06T10:01:03.409000Z TestFramework (INFO): Mining a big blockchain...
2024-08-06T10:01:04.554000Z TestFramework (INFO): Disk Usage is: 17 height: 751
2024-08-06T10:03:44.066000Z TestFramework (INFO): Disk Usage is: 17 height: 2751
2024-08-06T10:07:05.624000Z TestFramework (INFO): Disk Usage is: 17 height: 4751
2024-08-06T10:11:33.505000Z TestFramework (INFO): Disk Usage is: 33 height: 6751
2024-08-06T10:17:34.702000Z TestFramework (INFO): Disk Usage is: 33 height: 8751
2024-08-06T10:18:30.697000Z TestFramework (INFO): Final Usage is: 33 height: 9000
```